### ISSUE TITLE
[Feature Request] Book completion percentage

### ISSUE THREAD

**Issue by prince-modi** [LeoKlaus/plappa#168]
It would be nice to have a way to see % of book completed somewhere. Maybe in the currently playing tab or under the resume button on the book info tab.

**Comment by MadWalnut** [comment]
Already live on TestFlight: #146

**Comment by LeoKlaus** [comment]
Thank you for pointing that out @MadWalnut!

The way it currently works in the TestFlight version shows the total remaining time (for example 6h 42m), not a percentage.
Would this suffice for you @prince-modi or would you prefer a percentage?

**Comment by mrburns-42** [comment]
I would also find the percentage display useful, but in addition to the remaining time. I find the variant implemented in Prologue the most beautiful.

Here is a screenshot of it:

![image](https://github.com/user-attachments/assets/4c942f9d-04cd-4a94-b1e0-ae6d1f0baf97)

**Comment by mrburns-42** [comment]
I forgot to write that when you press on the blue field with the percentage, it disappears and the cover is completely visible.

**Comment by prince-modi** [comment]
@LeoKlaus yeah I am aware of the total time remaining thing. But as @mrburns-42 said, a place to see % completed would be nicer.

**Comment by mrburns-42** [comment]
Alternatively, you could also do this as in the ABS app:

![image](https://github.com/user-attachments/assets/3dad2826-2063-4758-8035-f0ce68315ceb)

**Comment by kevin-twingstrom** [comment]
I'd love to see this as well!  I miss the percentage from Prologue

**Comment by Brittca** [comment]
This would be a great quality of life feature. Any updates?
